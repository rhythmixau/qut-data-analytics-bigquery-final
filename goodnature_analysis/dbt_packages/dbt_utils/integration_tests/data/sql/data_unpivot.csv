version https://git-lfs.github.com/spec/v1
oid sha256:72b90acc4c7a7c8cc637812aa45d217c1045577d0934af745d3f2fabf7c3bbeb
size 152
