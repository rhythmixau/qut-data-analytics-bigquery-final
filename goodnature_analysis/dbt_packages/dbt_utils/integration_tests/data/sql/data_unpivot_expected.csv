version https://git-lfs.github.com/spec/v1
oid sha256:a24f3badb444235740a231fe13405acf25c40c5292d7da2ebb008d21ff97fb3c
size 246
